### DISCUSSION TITLE
"Vehicle Status", "Check Control Messages" in CLI

### DISCUSSION THREAD

**Discussion by HeraldCoupe** [bimmerconnected/bimmer_connected#511] [Feature request] [answered]
I use the CLI. The returned Json contains a huge amount of useful data but there is something extra that I would find useful.

On the main screen in the MyBMW app there is big white text "ALL GOOD" (unless there is a problem which has happened occasionally)
I would like to see this and the fields that are used to determine if all is good (or not) please.

Thank you, I really appreciate your work.

**Comment by rikroe** [comment, marked as answer]
While we don't have the "ALL GOOD" text, you can check the following two booleans:
- `condition_based_services.is_service_required` will turn to `True` if you need a service (like oil change)
- `check_control_messages.has_check_control_messages` will turn to `True` if there is any message with a severity >= medium.

Don't know how the app will change depending if there is a message though.
